portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -234,6 +234,8 @@ definition {
 		Click(
 			key_text = "Publish",
 			locator1 = "Button#ANY");
+
+		Alert.viewSuccessMessageText(successMessage = "Success:Variant name was published successfully.");
```

Comment:
Hey @manuferreira, I thought you could use the message that way. I think with this way the message can be used in cases where the control or variant was published and you could not use it within the test as in this case [here](https://github.com/liferay-ac/liferay-portal/pull/729/commits/e06cb3cacf72f34e6ae27833486ba60c8e9c7a97#diff-e151d3c674453d6b7dd8eb80ebc82d09d434b8cca73e0b460a72b6b78934c0f3R239-R241). What do you think?

`Alert.viewSuccessMessageText(successMessage = "Success:${variantName} was published successfully.");`

Reply:
Yes, you are correct I'm going insane with the AB test :sob: lol

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/0ec306da514d1bb61801c6049b0c126b5ff8ce88